Source/WTF/wtf/text/CStringView.h

```diff
@@ -70,17 +66,15 @@ class CStringView final {
         m_spanWithNullTerminator = byteCast<char8_t>(literal.spanIncludingNullTerminator());
     }
 
-    unsigned hash() const;
     bool isNull() const { return m_spanWithNullTerminator.empty(); }
 
     // This method is designed to interface with external C functions handling UTF8 strings. Interactions with other
```

Comment:
We normally don't use the word method since C++ calls,them functions or member functions.